.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7
+  hmpps: ministryofjustice/hmpps@7.3.3
```

Comment:
better to just pin at 7 instead which will float to latest

Reply:
@petergphillips sorted